core/client/fs/src/main/java/alluxio/client/block/stream/DefaultBlockWorkerClient.java

```diff
@@ -102,14 +102,12 @@ public DefaultBlockWorkerClient(UserState userState, GrpcServerAddress address,
         mStreamingChannel = GrpcChannelBuilder.newBuilder(address, alluxioConf)
             .setSubject(userState.getSubject())
             .setNetworkGroup(GrpcNetworkGroup.STREAMING)
-            .setClientType("DefaultBlockWorkerClient-Stream")
```

Comment:
I guess client type helps us debug e.g. give info in related thread name?
will we be able to see similar information?

Reply:
I don't really understand the logic of passing extra information for "debugging". If client type does not change the behavior of this class, why do you need that to debug this class? I'm assuming it's used to debug something else? Then why do we need to add information here to debug something else?

Reply:
Couldn't find a place where it's used for debugging or logging,,,,,  NVM

Reply:
https://github.com/Alluxio/alluxio/pull/9747 it is used for tracing but I think Rongrong is right that client type is meaning less since it doesn't affect behavior...

Reply:
After understand how this whole GRPC channel stuff work this makes even less sense. We create one ManagedChannel per GrpcConnectionKey(which has a GrpcChannelKey), and GrpcChannelKey does not use client type when compute equals/hashcode, which means, different client with the same server address and network group gets the same GrpcChannelKey, which will map to the same channel. So if you ever use this information to debug anything, you might be sent to the wrong direction... 🤦‍♀️ 🤷‍♀️